flake.nix

```diff
@@ -6,15 +6,9 @@
     haskellNix.url = "github:input-output-hk/haskell.nix";
     utils.url = "github:numtide/flake-utils";
     iohkNix = {
-      url = "github:input-output-hk/iohk-nix";
+      url = "github:input-output-hk/iohk-nix?ref=angerman/remove-old-envs";
```

Comment:
We should have input-output-hk/iohk-nix#534 merged _before_ this is merged, and _this_ line should be reverted, followed by a `nix flake lock --update-input iohkNix`.